pkg/security/tests/syscall_tester/c/syscall_tester.c

```diff
@@ -1253,6 +1312,7 @@ int main(int argc, char **argv) {
         int sub_argc = last_arg - i;
         char **sub_argv = argv + i;
         int exit_code = 0;
+        printf("Yihaaaaaa\n");
```

Comment:
Yoohoo ! Not an issue during review, but before merging don't forget to remove those !